nemo/core/optim/lr_scheduler.py

```diff
@@ -232,15 +337,27 @@ def _get_lr(self, step):
                     f"{self} received an initial learning rate that was lower than the minimum learning rate."
                 )
 
-        new_lrs = [
-            _cosine_annealing(
```

Comment:
CosineAnnealing needs to extend WarmupAnnealHoldPolicy(_LRScheduler) + add the two new args and pass them to constructor